test/test_path.py

```diff
@@ -396,7 +396,12 @@ def test_local_path_replace_path(local_path):
         if os.path.isdir(f'{TEMP_DIR}/{dirname}'):
             os.rmdir(f'{TEMP_DIR}/{dirname}')
 
-
+@pytest.mark.xfail(reason="This test is currently broken, but this is the behavior irodsConnector.dataOperations.DataOperation.download_data relies on")
```

Comment:
The new changes don't work on my platform, I've first added the broken test and will look into solving it

Reply:
It passes on my macOS and Windows machines.  Now, both with _and_ without the decorator.